<launch>

  <arg name="target_frame" default="odom" />

  <node pkg="orkobj_tomoveit" type="orkobj_tomoveit" name="orkobj_tomoveit" 
    args=" --target_frame=$(arg target_frame)" output="screen">
    <param name="table_topic" value="/table_array" />
   <param name="object_topic" value="/recognized_object_array" />
   <param name="target_frame" value="odom" />
  </node>

</launch>
